### ISSUE TITLE
Backup restore locks projects and shows no existing components

### ISSUE THREAD

**Issue by JMora77** [WeblateOrg/weblate#10852]
### Describe the issue

I was testing the borg backup feature in weblate and upon restoring the backup I noticed that the projects had the label that indicated that the project is locked (See Image). I see that the information is all there in the backup but it is not shown in the app.

I have found little to no information online regarding such an issue.

### I already tried

- [X] I've read and searched [the documentation](https://docs.weblate.org/).
- [X] I've searched for similar filed issues in this repository.

### Steps to reproduce the behavior

The issue happened after following the directions for a borg backup and restore in the weblate documentation.

### Screenshots

![Screenshot 2024-01-25 091318](https://github.com/WeblateOrg/weblate/assets/157685480/c19d28f1-b7f7-42e4-bb2d-6ec617733388)

### How do you run Weblate?

Docker container

### Weblate versions

 weblate/weblate:edge
Hash: sha256:780310018d21eba2ef5e06b66acab7bf2e86b3ed1778640707ad47d0525cb213.

**Comment by nijel** [comment]
* Maybe there were some errors shown when restoring the database? Did you follow the process?
* Were any components restored? Empty projects showed as locked until 8470b0531a5ea64387ae0c45fb5e62639407c515.
* Is there any reason for locking shown in the component history?

**Comment by JMora77** [comment]
Yes, while restoring the database there were errors regarding items that already existed or duplicate ids. 

Also, I followed the instructions in the Weblate docs. All projects were restored but no components were.

**Comment by nijel** [comment]
There shouldn't be any errors. And definitely you cannot expect full restore in case of errors. Can you please share log from the restoring?

**Comment by JMora77** [comment]
Ok, I will check the k8s configuration to see if some data persisted after the restart.

I have attached the logs from the psql pod after restoring. 

[weblate-postgresql-0_postgresql.log](https://github.com/WeblateOrg/weblate/files/14055664/weblate-postgresql-0_postgresql.log)

**Comment by JMora77** [comment]
I solved the issue by deleting the weblate database, creating a new one and then restoring it.

**Comment by nijel** [comment]
Thanks for the followup, https://github.com/WeblateOrg/weblate/pull/10868 should make the backups behave better in such cases.

**Comment by JMora77** [comment]
Thank you. Keep up the good work!
